index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&family=Work+Sans:wght@500&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header id="aboutme">
+      <h1>Andrea Carlota</h1>
+      <p>
+        I’m a Graphic Designer from Caracas, Venezuela interested in design,
+        fashion, illustration, architecture and getting the work done.
+      </p>
+      <ul class="contact-info">
+        <li class="contact-info">agonzaleznoriega@mica.edu</li>
+        <li class="contact-info">andreacarlotagn@gmail.com</li>
```

Comment:
For an HTML resume, it also makes sense for these addresses to be links; you do that by using [`mailto`](https://css-tricks.com/snippets/html/mailto-links/):
```suggestion
        <li class="contact-info"><a href="mailto:agonzaleznoriega@mica.edu">agonzaleznoriega@mica.edu</a></li>
        <li class="contact-info"><a href="mailto:andreacarlotagn@gmail.com">andreacarlotagn@gmail.com</a></li>
```